content/documentation/glossary/concepts/index.adoc

```diff
@@ -0,0 +1,105 @@
+
+=== Application Name
+
+It's the name of the <<Application>> deployed in your environment. This name is provided by the <<Label App>> on the <<Workload>>
+
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
+
+
+== Istio Config
+
+It's a configuration object in a Istio environment
+
+
+=== Istio object/configuration Type
+
+It's the type of the <<Istio Config>>, could be <<Gateway>>, <<VirtualService>>, <<DestinationRule>>, <<ServiceEntry>>, <<Rule>>, <<QuotaSpec>> or <<QuotaSpecBinding>>.
+
+== Istio Sidecar
+
+Read the <<Istio Sidecar>> definition in link:https://istio.io/docs/reference/commands/sidecar-injector/[Istio Sidecar].
+
+== Quota
+
+A limited or fixed number or amount of resources.
+
+== QuotaSpec
+
+File that define a <<Quota>>
+
+== QuotaSpecBinding
+
+Quota spec binding is used to bind link:https://istio.io/help/faq/mixer/[mixer] quota resource name to a proxy.
+
+== Label
+
+It's a tag. The client/user can identify a set of objects with this.
+
+An empty link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[label selector] (that is, one with zero requirements) selects every object in the collection.
+
+A null link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[label selector] (which is only possible for optional selector fields) selects no objects.
+
+For example, Istio use the <<Label App>> & <<Label Version>> in a <<Workload>> to specify the version and the application.
+
+=== Label App
+
+This represents the app label of an object. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Labels Requirements]
+
+=== Label Version
+
+This represents the version label of an object. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Labels Requirements]
+
+
+== Namespace
+
+Namespaces are intended for use in environments with many users spread across multiple teams, or projects
+
+Namespaces are a way to divide cluster resources between multiple users
+
+== ReplicaSet
+
+Ensures that a specified number of pod replicas are running at any one time
+
+== Service
+
+Pods are dynamically, they are mortal. A <<Service>> is an abstraction which defines a logical set of Pods and a policy by which to access them.
+A Service is determined by a <<Label>>
+
+=== Service Entry
+
+Read the <<Service Entry>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#ServiceEntry[Istio Service Entry]
+
+
+== VirtualService
+
+Read the <<VirtualService>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#VirtualService[Istio VirtualService]
+
+== Workload
+
+Read the <<Workload>> definition in link:https://istio.io/help/glossary/#workload[Istio Glossary]
+
+=== Workload Type
+
+It's the type of the <<Workload>>, now we only support <<Deployment>> type.
```

Comment:
Not sure we want to state what we support in the Terminology section.  But if so: 

", now we only support <<Deployment>> type." -> ".  Kiali currently supports the <<Deployment>> type."